scripts/irods/upgrade_configuration.py

```diff
@@ -40,7 +40,7 @@ def requires_upgrade(irods_config):
 def upgrade(irods_config):
     l = logging.getLogger(__name__)
 
-    if not os.path.exists('.'.join([paths.version_path(), 'previous'])) and os.path.exists(os.path.join(paths.irods_directory(), 'VERSION.previous')):
+    if not os.path.exists('.'.join([paths.version_path(), 'previous'])) and os.path.exists(os.path.join(paths.irods_directory(), 'VERSION.previous')): # TODO Investigate
```

Comment:
No change needed here because it is specific to legacy versions of iRODS.

Notice the `VERSION.previous` reference.